query-connector/src/app/constants.ts

```diff
@@ -81,6 +81,7 @@ export const FhirServers = [
   "JMC Meld: Direct",
   "JMC Meld: eHealthExchange",
   "Public HAPI: Direct",
+  "Local e2e HAPI Server: Direct",
```

Comment:
Are we now going to display this as an option in the FHIR server drop down in the UI? I think we probably want to avoid that

Reply:
Could maybe do a check for whether this is at the live domain and filter it out if it is

Reply:
Yeah, it's currently being displayed. I couldn't figure out a way to make it work another way on the e2e tests besides selecting the server from the drop down. I figured that it was behind an "Advanced" label and I haven't seen folks attempting to query other servers, it would be okay. 

My inclination would be to change the FHIR server option within the tests. I'm not sure how that would be done exactly, but I can look to see if it's possible in playwright.

Reply:
Ehh maybe this isn't worth the hassle atm